front/src/modules/trainschedule/components/TimetableV2/hooks.ts

```diff
@@ -88,15 +89,14 @@ const useTrainSchedulesDetails = (
 
         // Apply scheduled points honored filter
         if (scheduledPointsHonoredFilter !== 'both') {
-          if (trainSummary.status === 'success') {
-            const isHonored = trainSummary.scheduled_points_honored;
-            if (
-              (scheduledPointsHonoredFilter === 'honored' && !isHonored) ||
-              (scheduledPointsHonoredFilter === 'notHonored' && isHonored)
-            ) {
-              return false;
-            }
-          } else {
+          if (trainSummary.status !== 'success') return false;
+
+          const isHonored = isScheduledPointsNotHonored(trainSchedule, trainSummary);
```

Comment:
I think isNotHonored would be a better name ;)
```suggestion
          const isNotHonored = isScheduledPointsNotHonored(trainSchedule, trainSummary);
```

Reply:
Done